cli/setup.py

```diff
@@ -125,7 +125,7 @@ def find_executable(env_name, exec_name):
     "ruamel.yaml>=0.16.0,<0.18",
     "tqdm~=4.46",
     "packaging~=21.0",
-    "jsonschema~=3.2",
+    "jsonschema~=4.6.1",
```

Comment:
```suggestion
    "jsonschema~=4.6",
```
So minor version bumps are allowed?

Reply:
Sounds good.